performance-test/scripts/Resident_Test_script.jmx

```diff
@@ -0,0 +1,8333 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
+      <boolProp name="TestPlan.functional_mode">false</boolProp>
+      <boolProp name="TestPlan.tearDown_on_shutdown">true</boolProp>
+      <boolProp name="TestPlan.serialize_threadgroups">false</boolProp>
```

Comment:
S03 Manage My VID (Execution) -> temp_vid - variable naming convetion to be corrected

Reply:
${__jexl3("${temp_vid}" == "O")} - this condition check is also confusing

Reply:
1. Have changed the variable name as per naming convention.
2. ${__jexl3("${temporaryVid}" == "EMP")} - new condition added.
3. Constant timer under revoke vid is removed.

Reply:
EMP and NOT_FOUND both sounds as same condition.

Reply:
${__jexl3("${temporaryVid}" == "TEMPORARY")} - new expression added

Reply:
Changes are updated as per inputs.